ai-verify-portal/src/modules/projectTemplate/canvas.tsx

```diff
@@ -459,7 +463,12 @@ export default function CanvasComponent(props: CanvasProps) {
   
   const onDropDragOver = (e: DragOverEvent) => {
     if (!dragItem) return;
-    return { w: dragItem.widgetSize.minW, h: dragItem.widgetSize.minH }
+    if (dragItem.dynamicHeight) {
+      // return { w: dragItem.widgetSize.minW, h: GRID_MAX_ROWS }
+      return { w: dragItem.widgetSize.minW, h: dragItem.widgetSize.minH }
+    } else {
+      return { w: dragItem.widgetSize.minW, h: dragItem.widgetSize.minH }

```

Comment:
Originally wanted to set to GRID_MAX_ROWS so that it automatically fills up to the rest of the canvas but realizes that by doing to this way I am unable to set the y to any position. So I changed to 'autoexpand' the height on drop instead